CRM/Utils/System/WordPress.php

```diff
@@ -355,6 +355,20 @@ public function permissionDenied() {
   }
 
   /**
+   * Determine the native ID of the CMS user.
+   *
+   * @param string $username
+   * @return int|NULL
+   */
+  public function getUfId($username) {
+    $userdata = get_user_by('login', $username);
```

Comment:
@seamuslee001 is there a reason you want to use get_user_by() as opposed to get_current_user_id() ?

Reply:
@kcristiano I saw its what is used in the function just above in L334, The purpose of this i think is that in the E2E tests we are passing through a user name + pw and authenticating then wanting to get the id back

Reply:
You are correct - I was not thinking about the proper context.  As we are running tests we need to pas in the user name = pw.